media/redesign/stylus/components/wiki/revision-list.styl

```diff
@@ -97,3 +97,7 @@ html[dir=rtl] .revision-list-controls:before {
     width: 460px;
 }
 
+/* make the Compare selected revisions button align to layout */
+.revision-list-controls .fixed {
+    bidi-style(left, 24px, right, 24px);
```

Comment:
The 4th parameter needs to be the inverse value for left, in this case "auto".

The way the bidi mixin works is that the first two values are compiled into the regular style sheet:
```
.revision-list-controls .fixed {
   left: 24px;
}
```
And then an over-ride is made for the RTL text, over-riding the value of the first property with the value provided in the 4th parameter.
```
html[dir='rtl'] .revision-list-controls .fixed {
   left: auto;
   right: 24px;
}
```

Thanks for the quick turn around on my comments!